tools/reflection_generator/java_method.py

```diff
@@ -415,11 +416,12 @@ def FormatSingleParam(self, param_type, param_name, param_string_type):
       # the way bridge uses as the condition for whether call super or
       # call wrapper in override call
       #   XWalkViewInternal view => (view instanceof XWalkViewBridge)
-      if (is_internal_class and
-          not java_data.HasInstanceCreateInternallyAnnotation()):
-        return'(%s instanceof %s)' % (param_name, java_data.GetBridgeName())
-      else:
-        return None
+      if is_internal_class:
+        if not java_data.HasInstanceCreateInternallyAnnotation():
+          return'(%s instanceof %s)' % (param_name, java_data.GetBridgeName())
+        else:
+          self._parameter_is_internal = True
+      return None
```

Comment:
sorry, I don't understand here, what's the different between `CustomViewCallbackHandlerInternal&CustomViewCallbackInternal` and `ClientCertRequestHandlerInternal & ClientCertRequestInternal` that need you change the java_method.py?